config/crd/bases/operator.authorino.kuadrant.io_authorinos.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.1
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
The Makefile says v0.6.1. This change is probably due to an older version installed locally.

https://github.com/Kuadrant/authorino-operator/blob/c5c0510c810f94a8267da238705799ea1af32b6e/Makefile#L134